packages/tables/resources/views/components/cell.blade.php

```diff
@@ -1,12 +1,13 @@
 @props([
-    'action' => null,
-    'name',
-    'record',
-    'shouldOpenUrlInNewTab' => false,
-    'url' => null,
+'action' => null,
+'name',
+'record',
+'shouldOpenUrlInNewTab' => false,
+'url' => null,
+'alignment' => null,
 ])
 
-<td {{ $attributes->class(['filament-tables-cell', 'dark:text-white' => config('tables.dark_mode')]) }}>
+<td {{ $attributes->class(['filament-tables-cell', 'dark:text-white' => config('tables.dark_mode'), "text-$alignment" => $alignment]) }}>
```

Comment:
This won't work with Tailwind since it can't accurately determine what classes should be kept/removed in the purge. You'll have to use a match statement or `Arr::toCssClasses` instead. There's plenty of examples in the code base.

Reply:
it's now correctly?